core/editor/src/tools/ellipse.rs

```diff
@@ -69,19 +67,24 @@ impl Fsm for EllipseToolFsmState {
 
 				EllipseToolFsmState::LmbDown
 			}
-
 			(EllipseToolFsmState::LmbDown, Event::LmbUp(mouse_state)) => {
 				data.drag_current = mouse_state.position;
 
 				operations.push(Operation::ClearWorkingFolder);
+				// TODO - introduce comparison threshold when operating with canvas coordinates (https://github.com/GraphiteEditor/Graphite/issues/100)
 				if data.drag_start != data.drag_current {
 					operations.push(make_operation(data, tool_data));
 					operations.push(Operation::CommitTransaction);
 				}
 
 				EllipseToolFsmState::Ready
 			}
+			// TODO - simplify with or_patterns when available in stable rust (https://github.com/rust-lang/rust/issues/54883)
```

Comment:
Didn't you say this will be available in Rust 1.53? Since we have an actual release number, it's probably clearer to future readers if we just write the version number since that closed issue doesn't even indicate the relevant status unless you go digging.